src/components/CardItem.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class CardItem extends Component {
+  render() {
+    const { products } = this.props;
+    return (
+      <div>
+        {products.map(({ title, price, thumbnail, id }) => (
+          <div data-testid="product" key={ id }>
+            <h2>{ title }</h2>
+            <span>{ price }</span>
+            <img src={ thumbnail } alt="alt" />
+          </div>
+        ))}
```

Comment:
O que acham de deixar  a responsabilidade de renderiza ar os produtos ao componente pai (Home), e o componente `CardItem` recebendo a props de cada produto? Desse jeito, ao ler o componente pai, conseguimos  saber com facilidade de onde a lista de produtos está surgindo.
```suggestion
    const { title, price, thumbnail, id } = this.props;
    return (
        <div data-testid="product" key={ id }>
          <h2>{ title }</h2>
          <span>{ price }</span>
          <img src={ thumbnail } alt="alt" />
        </div>
       )
```

Reply:
Concordo com a alteração Sérgio, ficaria bem melhor estruturado. Podemos trabalhar nisso hoje. :)